content/guides/exporters/supported-exporters/Python/ApplicationInsights.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Azure Monitor"
+date: 2018-10-15
+draft: false
+weight: 3
+class: "resized-logo"
+aliases: [/supported-exporters/python/applicationinsights]
+logo: /img/partners/microsoft_logo.svg
+---
+
+- [Introduction](#introduction)
+- [Creating the exporter](#creating-the-exporter)
+- [Viewing your traces](#viewing-your-traces)
+
+## Introduction
+Azure Monitor is an extensible Application Performance Management (APM) service for web developers on multiple platforms. Offered by Microsoft Azure, it's a complete at-scale telemetry and monitoring solution.
+
+If you don't have an Azure account yet, [click here](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-overview) to start.
+
+## Creating the exporter
```

Comment:
should it be "register" or "configure"? instead of "creating"?

Reply:
That's settled wording, every exporter page has this. I don't know if I love it, but at least it's consistent so that people understand the correlation.